conf/ocsci/vsphere_use_custom_sc_in_deployment.yaml

```diff
@@ -0,0 +1,6 @@
+---
+# This configuration file is used for using custom storage class ( thin-csi-odf ) in ODF deployment.
+# thin-csi-odf is clone of thin-csi but with storage policy as "vSAN Default Storage Policy"
+# This yaml file is used to save the storage in vSphere environment.
+ENV_DATA:
+  use_custom_sc_in_deployment: true
```

Comment:
I would go other way, to have this as default option in default config of ocs-ci, and if someone would like to have it otherwise they can use config file to disable that and use the default from product.